pandas/tests/frame/test_quantile.py

```diff
@@ -3,24 +3,23 @@
 
 import pandas as pd
 from pandas import DataFrame, Series, Timestamp
-from pandas.tests.frame.common import TestData
 import pandas.util.testing as tm
 from pandas.util.testing import assert_frame_equal, assert_series_equal
 
 
-class TestDataFrameQuantile(TestData):
+class TestDataFrameQuantile:
 
-    def test_quantile(self):
+    def test_quantile(self, datetime_frame):
         from numpy import percentile
 
-        q = self.tsframe.quantile(0.1, axis=0)
-        assert q['A'] == percentile(self.tsframe['A'], 10)
-        tm.assert_index_equal(q.index, self.tsframe.columns)
+        q = datetime_frame.quantile(0.1, axis=0)
```

Comment:
a pattern that is now being used across the codebase is to 'rename' the fixture at the beginning of the test (it facilitates the use of more descriptive fixture names)

 so just add `df = datetime_frame` at the beginning of the test and then use `df` throughout the test.